articles/azure-app-configuration/enable-dynamic-configuration-dotnet-core-push-refresh.md

```diff
@@ -41,7 +41,9 @@ In this tutorial, you learn how to:
 
```

Comment:
Can you please help to remove line 29? Its content is duplicated from the "Push Model" description. The reason it listed for choosing Service Bus is actually misleading.

Reply:
removed